### ISSUE TITLE
Bug Report: Unable to Import Standalone Notes Without Parent Item

### ISSUE THREAD

**Issue by Jarartur** [PKM-er/obsidian-zotlit#397]
First, thank you for this fantastic plugin!

I’ve encountered an issue with importing standalone notes created in Zotero using the "Create Note From Annotations" option (when selecting multiple elements). Specifically:

    Standalone notes without a parent item cannot be selected for import using the "ZotLit: Import Note" command in Obsidian.
    However, if the standalone note is attached to a random item, it becomes importable.

It would be helpful to support importing standalone notes directly, even without a parent item.

(Also, great job on how the imported notes correctly link to references—very impressive!)

**Comment by Servinjesus1** [comment]
@Jarartur What template code do you use to import the attached note? I am seeing the note under the notes field, but when importing, `it.notes` is an empty list. Some troubleshooting code I can put in my template includes:

```rust
<%= JSON.stringify(it.notes) %>
<%- if (it.notes.length) { %>
---

## Notes

<% for (const note of it.notes) { %>
<%= "note" %>
<%= note -%>
<%= note.content.replace(/(#+)/g, "##$1")-%>
<%- } %>
<%- } %>
```

The stringify returns `[]`, which is apparently not False-like in javascript, hence the need for `it.notes.length` in the following line, which prevents parsing of the remaining lines.

Yet, in the item browser I definitely have one note in the item so IDK what's going on.
<img width="528" alt="image" src="https://github.com/user-attachments/assets/e1fa0bb4-cdee-4ce8-846f-f9b90df62880" />

**Comment by Jarartur** [comment]
I don't think it uses any template at all tbh. When I import it like that it just creates a note (without any properties) containing all the text that was in the zotero note.

**Comment by tinix84** [comment]
Reproducing on Zotero 9.0.3 + ZotLit 1.1.12 — orphan PDF attachments (no parent item) are completely invisible to `ZotLit: Open quick switcher`, even though they're queryable in the SQLite (`itemAttachments` where `parentItemID IS NULL`). Cross-linking from #471.

**Comment by aidenlx** [comment]
Zotero's Obsidian menu now has a dedicated way to import notes, and it recognizes a Zotero note as importable on its own, with no parent item required. Standalone notes created from annotations, like the one you described, should import directly now.

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
